build/lib/src/analyzer/resolver.dart

```diff
@@ -26,7 +27,9 @@ abstract class Resolver {
   /// * Throws [NonLibraryAssetException] if [assetId] is not a Dart library.
   Future<LibraryElement> libraryFor(AssetId assetId);
 
-  /// Returns the first library identified by [libraryName].
+  /// Returns the first resolved library identified by [libraryName]. A library
```

Comment:
[nit] Blank line after the first sentence.

https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph